test/big-upload.js

```diff
@@ -0,0 +1,111 @@
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const qs = require('qs');
+
+const t = require('../test-lib/test.js');
+const bigUpload = require('../lib/big-upload-client.js');
+
+const buffer = fs.readFileSync(path.resolve(__dirname, 'data/upload_tests/crop_image.png'));
+
+describe('Big Upload', function() {
+
+  let apos;
+  let received = false;
+
+  after(async function () {
+    return t.destroy(apos);
+  });
+
+  this.timeout(t.timeout);
+
+  it('init apos', async function() {
```

Comment:
apos initialization should be made from a `before` function in order to make tests independent ("onlyable")